packages/codec/lib/format/utils/inspect.ts

```diff
@@ -23,6 +23,10 @@ function cleanStylize(options: InspectOptions): InspectOptions {
   return clonedOptions;
 }
 
+export interface ResultInspectorOptions {
+  noHideAddress?: boolean;
```

Comment:
If this appears in the docs, we might want to add a doc comment about this.

Reply:
Ooh, good point.